### ISSUE TITLE
directory selector

### ISSUE THREAD

**Issue by sancelot** [funkizer/ng-file-manager#3]
Hi,
In my application, I need being able to use a filedirectory selector in angular 6. (it is not defined with which versions it will run)

Is this possible with this api ?
Regards,
steph

**Comment by SpeedoPasanen** [comment]
Yes, if not out of the box, at least by writing a custom connector. But I haven't had the time to maintain this in months, don't know if this will even work with Angular 6. Sorry, just can't find the time with too much day work.
